src/orchestrator.rs

```diff
@@ -81,6 +82,10 @@ impl Orchestrator {
         Ok(orchestrator)
     }
 
+    pub fn get_config(&self) -> OrchestratorConfig {
+        self.ctx.config.clone()
+    }
```

Comment:
Agreed,  it should be renamed to `config()` and return a reference.

```suggestion
    pub fn config(&self) -> &OrchestratorConfig {
        &self.ctx.config
    }
```